### ISSUE TITLE
Add a "default-username" setting

### ISSUE THREAD

**Issue by woodruffw** [woodruffw/kbs2#199]
Most websites use the user's email as their username these days, so it might be nice to have a setting like this:

```toml
[commands.new]
default-username = "foo@bar.com"
```

When set, this should have the same behavior as `kbs2 new` with a generator: if the user doesn't enter a username, then the default one is substituted.

**Comment by woodruffw** [comment]
This is slightly annoying to do with how I've designed the current `kbs2::input` module. That should probably be refactored first.

**Comment by woodruffw** [comment]
This will be much easier to implement once #306 lands.
